### DISCUSSION TITLE
Data limits

### DISCUSSION THREAD

**Discussion by lunchboxer** [beginner-corp/begin-community#71] [Support] [answered]
The homepage lists a free tier as offering:

"Limited throughput and database usage"

To avoid any inconvenient surprises, I'd like to know what those limits are.

**Comment by brianleroux** [comment, marked as answer]
Hey @lunchboxer thanks for this! We need to do a better job explaining the free tier limits. Currently we host roughly 1000 apps in our free tier per AWS account. So you can suffer throttles from a noisy neighbor. In our paid tiers apps are deployed to a completely isolated AWS account so normal service quotas apply. 

The limits and quotas for the following services can be found here:
- Lambda https://docs.amazonaws.cn/en_us/lambda/latest/dg/gettingstarted-limits.html
- API Gateway https://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html
- DynamoDB https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html

LMK if this helps or if you need further clarity. Cheers!
